.github/workflows/convert.yml

```diff
@@ -0,0 +1,53 @@
+name: build nbconvert_html5 and export samples
+
+on: 
+  push:
+    branches-ignore:
+      - gh-pages
+    paths:
+      - .github/workflows/**
+      - nbconvert_html5/**
+      - "pyproject.toml"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.10"]
+
+    steps:
+      - uses: actions/checkout@v3
+        name: Fetch all history for all tags and branches
+        with:
+          fetch-depth: 0
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: 'pip'
+          cache-dependency-path: |
+            pyproject.toml
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install hatch
+      - name: Convert
+        run: |
+          python -m hatch run task:convert
+      - name: Build the docs
+        run: |
+          python -m hatch run docs:build
+      - name: Deploy main 🚀
+        uses: JamesIves/github-pages-deploy-action@v4
```

Comment:
recall you can do all of this with the built-in GHA options, not... whatever this is.

Reply:
okay. cool. been using this for a real long. ill find the more appropirate action